rest-api/tools/import_organizations.sh

```diff
@@ -42,12 +42,16 @@ GEOCODE_FLAG=--geocode_flag
 get_geocode_key
 fi
 
+EXTRA_ARGS="$@"
 if [ "${PROJECT}" ]
 then
+  echo "Getting credentials for ${PROJECT}..."
   source tools/auth_setup.sh
   run_cloud_sql_proxy
   set_db_connection_string
+  EXTRA_ARGS+=" --creds_file ${CREDS_FILE} --instance ${INSTANCE}"
 else
+  PROJECT='localhost'
```

Comment:
What is this for?

We normally don't expect PROJECT to be set (or passed in on the command line) when dealing with localhost. Setting it here may screw other things up. I think you're going to instead want to make --project a part of EXTRA_ARGS above and not set it here.

Reply:
changed logic to make project optional and handle localhost different now.